public/docs/ts/latest/guide/component-communication.jade

```diff
@@ -0,0 +1,707 @@
+  At the parent side `$event` represents the argument of the emitted event.
+  
+  [Jump to details](#-2-child-to-parent-communication-with-event-emitting)
+
+  __#3: Child to parent communication with direct method calling__
+  
+  Nothing prevents us to implement our parent so that it passes a reference to itself to its children, and those use that reference
+  to access their parent's methods and properties. When the parent embeds its children as content markup, we can hand the parent reference to 
+  children in a trivial way like this:
+  
++makeExample('component-communication/ts/fragments/parent-reference-1.html')
+  
+:marked
+  Beside this very technique, there are other ways to pass the parent reference to children, with little tricks we can even use dependency injection
+  to inject the parent into the child component's constructor.
+  The concrete technique does not matter: **Angular discourages this model, because it can sidestep the change detection mechanism, and so it may break
+  the data flow model.**
+  
+  [Jump to details](#-3-child-to-parent-communication-with-direct-method-calling)
+
+  __#4: Duplex parent/child communication through a service object__
+  
+  Parent and child components share their responsibilities. In this sharing both component types have their own responsibilities including UI logic
+  and others. The parent and its children both can manage state information about the UI logic. Often, the majority of communication is about
+  managing the overall state information of this component fabric.
+  
+  In many cases the logic can be outsourced to a separate service that is the single source of truth, stores the state and provides operations to
+  access and alter the state. In such a scenario, parent and child components do not need to talk with each other directly, each of them 
+  communicates with the intermediary service:
+  
++makeExample('component-communication/ts/fragments/service-intro.ts', 'service')
+
+:marked
+  This service is injected into the parent and child components via Angular dependency injection (via component constructors):
+
++makeExample('component-communication/ts/fragments/service-intro.ts', 'components')
+
+:marked
+  Both component types can access the service instance injected into the constructor. The dependency injection mechanism takes care &mdash; 
+  unless we do not configure it in another way &mdash; that the parent and its child components use the same service instance.
+  We can design and implement the approprate service methods and properties, and access them from within the components. 
+  
+  [Jump to details](#-4-duplex-parent-child-communication-through-a-service-object)
+
+  __#5: Communication among unrelated components via a message bus__
+  
+  In many inter-component communication scenario, the communicating parties are not in a perent-child relation with each other. They can
```

Comment:
"parent" not "perent".  Rewrite anyway to avoid leading with a preposition. Perhaps simply:

"The communicating components aren't always in a parent-child relationship."